app/models/stacks_media_token.rb

```diff
@@ -55,7 +55,7 @@ def self.max_token_age
   def initialize(id, file_name, user_ip)
     @id = id
     @file_name = file_name
-    @user_ip = user_ip
+    @user_ip = last_ip_address(user_ip)
```

Comment:
if we do go with this approach, i'd prefer to rename `user_ip` to something more accurate like `user_ip_list` (or the verbose but even more accurate `user_ip_list_string`).

but my gut reaction is that this splitting logic should live in consumers of `StacksMediaToken`, and not `StacksMediaToken` itself, since the token class should (it seems to me) concern itself with the user's actual IP, and not the list of IPs that the caller gets from the request headers (that feels very controller-ish, since it's pretty specific to request handling).  that said, i wouldn't lose too much sleep over this living in `StacksMediaToken`, under the reasoning that all consumers of `StacksMediaToken` will want this functionality in practice.